src/components/enterprise-user-subsidy/enterprise-offers/EnterpriseOffersBalanceAlert.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import { Container, Alert, MailtoLink } from '@edx/paragon';
+import { WarningFilled } from '@edx/paragon/icons';
+import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
+import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+import { AppContext } from '@edx/frontend-platform/react';
+import {
+  LOW_BALANCE_CONTACT_ADMIN_TEXT,
+  LOW_BALANCE_ALERT_HEADING,
+  LOW_BALANCE_ALERT_TEXT,
+  NO_BALANCE_CONTACT_ADMIN_TEXT,
+  NO_BALANCE_ALERT_HEADING,
+  NO_BALANCE_ALERT_TEXT,
+  OFFER_BALANCE_CLICK_EVENT,
+} from './data/constants';
+
+const EnterpriseOffersBalanceAlert = ({ hasNoEnterpriseOffersBalance }) => {
+  const {
+    enterpriseConfig: { uuid: enterpriseCustomerUUID, adminUsers },
+  } = useContext(AppContext);
+
+  const adminText = hasNoEnterpriseOffersBalance ? NO_BALANCE_CONTACT_ADMIN_TEXT : LOW_BALANCE_CONTACT_ADMIN_TEXT;
+  const className = hasNoEnterpriseOffersBalance
+    ? 'no-offers-balance-alert-with-cta' : 'low-offers-balance-alert-with-cta';
+  const variant = hasNoEnterpriseOffersBalance ? 'danger' : 'warning';
+  const icon = WarningFilled;
+  const heading = hasNoEnterpriseOffersBalance ? NO_BALANCE_ALERT_HEADING : LOW_BALANCE_ALERT_HEADING;
+  const text = hasNoEnterpriseOffersBalance ? NO_BALANCE_ALERT_TEXT : LOW_BALANCE_ALERT_TEXT;
+
+  const adminEmails = adminUsers.map(user => user.email);
+  const hasAdminEmails = adminEmails.length > 0;
+
+  const { userId } = getAuthenticatedUser();
+  const actions = hasAdminEmails ? [
+    <MailtoLink
+      to={adminEmails}
+      target="_blank"
+      onClick={() => {
+        sendEnterpriseTrackEvent(
+          enterpriseCustomerUUID,
+          OFFER_BALANCE_CLICK_EVENT,
+          {
+            userId,
+          },
+        );
+      }}
+    >{adminText}
+    </MailtoLink>,
+  ] : [];
+
+  return (
+    <Container size="lg" className="pt-3">
+      <Alert
+        className={classNames({ [`${className}`]: hasAdminEmails })}
```

Comment:
Doubling back to the discussion of these class names, I'm still not understanding why we'd only want a class name here if the admin has emails. We're not referencing these class names anywhere else (for styles or testing), are we? If not, I'd be in favor of just removing the custom conditional class name here altogether? 🤔